en/core-libraries/collections.rst

```diff
@@ -1178,6 +1178,35 @@ places at the same time. In order to clone a collection out of another use the
         }
     }
 
+Creating your own custom Collections
+====================================
+
+It is possible to create your own collection classes in your application to store, manage and manipulate groups of items. Simply create your collection class and extend the core collection class instance.
+
+    namespace App\Lib\Collections;
+
+    use Cake\Collection\Collection;
+
+    class ExamplesCollection extends Collection
+    {
+
+    }
```

Comment:
Why would someone create a custom collection? We should give some context/use case that would require one.